gcloud-java-examples/src/main/java/com/google/gcloud/examples/bigquery/BigQueryExample.java

```diff
@@ -63,7 +63,7 @@
  * <li>login using gcloud SDK - {@code gcloud auth login}.</li>
  * <li>compile using maven - {@code mvn compile}</li>
  * <li>run using maven -
- * <pre>{@code mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.bigquery.BigQueryExample"
+ * <pre>{@code mvn exec:java -Dexec.mainClass="com.google.gcloud.example.bigquery.BigQueryExample"
```

Comment:
here and in other places. I assume the com.google.gcloud.examples.rename was not intentional?